### DISCUSSION TITLE
`vercel dev` not starting `@vercel/node` dev server.

### DISCUSSION THREAD

**Discussion by jensmeindertsma** [vercel/vercel#7110] [Help] [answered]
I've added a `api/index.js` file that just sends a "Hello, World" response. When I deploy my function to vercel, by running `vercel`, it works as expected. During `vercel dev` however, every request to `/api` or `/api/index` fails with a 502. Looking in my terminal: 
```
➜  vercel-sf vercel dev
Vercel CLI 23.1.2 dev (beta) — https://vercel.com/feedback
> Ready! Available at http://localhost:3000
Error! Failed to complete request to /api: Error: connect ECONNREFUSED ::1:49437
```
So then I tried to run `vercel dev --debug` and request the same URL:
```
> [debug] [2021-12-02T09:40:01.121Z] Requiring runtime: "@vercel/node"
> [debug] [2021-12-02T09:40:01.266Z] Adding build match for "api/index" with "@vercel/node@latest"
> [debug] [2021-12-02T09:40:01.266Z] Adding build match for "package-lock.json" with "@vercel/static"
> Ready! Available at http://localhost:3000
> [debug] [2021-12-02T09:41:01.616Z] Proxying to "@vercel/node" dev server (port=49490, pid=1801)
Error! Failed to complete request to /api: Error: connect ECONNREFUSED ::1:49490
```
It looks like the `@vercel/node` dev server isn't functioning correctly? How can I solve this?

**Comment by styfle** [comment, marked as answer]
Hi @jensmeindertsma 

This was fixed in #7094 and you can try it out today on the canary channel: `npm i -g vercel@canary`.

**Reply by jensmeindertsma** [reply]
Awesome!
